app/serializers/search/nested_user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module Search
+  class NestedUserSerializer
+    include FastJsonapi::ObjectSerializer
+
+    attributes :username, :name, :id, :profile_image_90, :pro
```

Comment:
should the `id` be the first attribute? I know order doesn't matter but it seems a bit of a convention with the other serializers

Reply:
I'm pretty in love with our organized mappings, let me spruce up these serializers.